src/DTF/Documents/Reference/dtfref.shfbproj

```diff
@@ -20,10 +20,13 @@
     <FooterText>&amp;lt%3bscript src=&amp;quot%3bhelplink.js&amp;quot%3b&amp;gt%3b&amp;lt%3b/script&amp;gt%3b</FooterText>
     <PresentationStyle>Prototype</PresentationStyle>
     <NamingMethod>MemberName</NamingMethod>
+	<FrameworkVersion>.NET Framework 4.6.2</FrameworkVersion> 
```

Comment:
We aren't targeting 4.6.2, we're targeting 3.5 IIRC. Are you able to build with the old SHFB if you set this to 3.5?